utilities/common.py

```diff
@@ -116,6 +116,9 @@ def common_parser(for_reco_highPU=False):
     parser.add_argument("--met", type=str, choices=["DeepMETReso", "RawPFMET"], help="MET (DeepMETReso or RawPFMET)", default="RawPFMET")                    
     parser.add_argument("-o", "--outfolder", type=str, default="", help="Output folder")
     parser.add_argument("-e", "--era", type=str, choices=["2016PreVFP","2016PostVFP"], help="Data set to process", default="2016PostVFP")
+    parser.add_argument("--customTag",
```

Comment:
This might make sense in setupCombineWMass instead since it isn't implemented for the low pu (though it could be)

Reply:
I would use 
```parser.add_argument("-p", "--postfix", type=str, help="Postfix for output file name", default=None)``` to stay in sync with the common_parser() 
then removing ```-p``` from ```"-p", "-pseudodata"``` in setupCombineWMass.py